artifactory/services/upload.go

```diff
@@ -496,7 +502,7 @@ type UploadData struct {
 
 type artifactContext func(UploadData) parallel.TaskFunc
 
-func (us *UploadService) createArtifactHandlerFunc(uploadResult *utils.UploadResult, uploadParams UploadParams) artifactContext {
+func (us *UploadService) createArtifactHandlerFunc(uploadResult *utils.Result, uploadParams UploadParams) artifactContext {
```

Comment:
It looks like the following local variables used in the function, do nor need to be declared implicitly. They can be created when returned from the functions returning them:
```
			var uploaded bool
			var target string
			var artifactFileInfo utils.FileInfo
```